builtin.aqua

```diff
@@ -383,3 +382,16 @@ service Stat("stat"):
 service Debug("debug"):
     -- Convert any object into a string
     stringify(o: ⊤) -> string
+
+-- Available only on rust peers
+service Worker("worker"):
```

Comment:
Would it be clearer to add such a definition?

I used it in my scripts to indicate that everything is happening in the root scope, but I'm unsure if it's a good idea for a lib.
```
service WorkerRoot("worker"):
  create() -> string
  get_peer_id() -> string
```